catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/StdidcAttribute.java

```diff
@@ -193,26 +201,29 @@ private StdidcAttribute(String longName, String shortName,
         return Collections.unmodifiableList(ATTRIBUTES);
     }
 
-    /**
-     * Get the alpha3 country code for and alpha2 country code.
-     * in alpha2 code.
-     * @param alpha2CountryCode the alpha2 country code.
-     * @return Returns null if null is passed in. If the alpha3 code lookup fails it will return the passed in alpha2 code. Otherwise returns the alpha3 country code.
-     */
-    private static String getAlpha3CountryCode(String alpha2CountryCode) {
-        if (alpha2CountryCode == null) {
-            return alpha2CountryCode;
+    private static Map<String, String> getFipsMap() {
+        Map<String, String> map = PropertiesLoader.getInstance()
```

Comment:
✏️ Might want to protect against the case where an admin has deleted the properties file.

Reply:
The PropertiesLoader protects against this. If it can't load the file it will just return an empty Properties object.